akka-actor/src/main/scala/akka/actor/ActorRefProvider.scala

```diff
@@ -532,7 +532,7 @@ private[akka] class LocalActorRefProvider private[akka] (
     override def stop(): Unit = {
       causeOfTermination.trySuccess(
         Terminated(provider.rootGuardian)(existenceConfirmed = true, addressTerminated = true)) //Idempotent
-      terminationPromise.tryCompleteWith(causeOfTermination.future) // Signal termination downstream, idempotent
+      terminationPromise.completeWith(causeOfTermination.future) // Signal termination downstream, idempotent
```

Comment:
why is this change needed? can we be sure that it hasn't already been completed?

Reply:
`tryCompleteWith` is deprecated in 2.13 because apparently `completeWith` always had these semantics already (unlike success/failure/complete): tested with `sbt ++2.11.12 console`:

```
import scala.concurrent._
p.completeWith(Future.successful(37))
p.completeWith(Future.successful(42))
implicit val ec = ExecutionContext.global
p.future.foreach(println)
37
```

Reply:
ah, my misunderstanding then

Reply:
yeah, very confusing, surprised me as well